ndmg/utils/bids_utils.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python
```

Comment:
I realized this is what happened with `ndmg_demo-dwi` above as well, but when you move things improperly it shows a bunch of old things as new, and makes it so you're the "blameable" one on Github rather than the original author. This happened with the deleted `ndmg/utils/bids.py` to this file. If you use `git mv` it makes it possible for me to actually review changes, and keeps me on the hook for my idiocy :)